test/JDBC/expected/BABEL_OBJECT_DEFINITION-vu-prepare.out

```diff
@@ -0,0 +1,57 @@
+CREATE DATABASE object_definition_db;
```

Comment:
I remember we have a test file for object definitions of various kind of objects, should we move these tests to that file instead if possible? We will not have to create new objects in that case.

Reply:
Those files are named specifically on the view names and there is no single file where all of these objects supported by object_definition() are created. IMO, it's better to have a separate test file.